src/containers/flows/MintDropNFT/steps/Info/Info.tsx

```diff
@@ -43,17 +43,21 @@ const Info = (props: InfoProps) => {
 
 	// If all data is loaded, and it's the public release
 	// or it's the whitelist release and the user is whitelisted
+
+	// Added to handle GENs drop
 	const isUserEligible =
 		!isUserDataLoading &&
 		!isAuctionDataLoading &&
-		(props.dropStage === Stage.Public ||
-			(props.dropStage === Stage.Whitelist &&
-				props.isUserWhitelisted &&
-				props.maxPurchasesPerUser! > 0));
+		(props.dropStage === Stage.Public || props.dropStage === Stage.Whitelist) &&
+		props.isUserWhitelisted &&
```

Comment:
I'm not sure if this is a requirement of the sale, the following code will check if the user is whitelisted even when the sale is public
```js
(props.dropStage === Stage.Public || props.dropStage === Stage.Whitelist) &&
props.isUserWhitelisted &&
```

I also see that `glow={props.isUserWhitelisted || props.dropStage === Stage.Public}` was changed to `glow={props.isUserWhitelisted}`.

Just double checking that this is the expected behaviour!

Reply:
here the public sale is nothing but a private mintlist sale.